txpool/account.go

```diff
@@ -172,6 +172,21 @@ func (a *account) setNonce(nonce uint64) {
 	atomic.StoreUint64(&a.nextNonce, nonce)
 }
 
+// Demotions returns the current value of demotions
+func (a *account) Demotions() uint64 {
```

Comment:
Why is this an exported method?

Reply:
No reason except for the reason to avoid name collision. I think it's no problem because this function just returns the value, it doesn't have side effect, and account is not exported